### PR TITLE
PCHR-2910: Fix notification tables

### PR DESCRIPTION
**Pull request by reneolivo**
This PR fixes an issue where tables inside notifications have white text against the notification's white background rendering the text invisible.

### Before
import contacts hr17 9000 2
* **Note**: the extra boder sticking out is because of an error in the template defined in CiviCRM:
https://github.com/civicrm/civicrm-core/blob/4.6/templates/CRM/Contact/Import/Form/DataSource.hlp#L97

### After
import contacts hr17 9000 1

### Technical description

The issue was first introduced when styling notifications for Shoreditch. In CiviCRM there is a special rule for tables inside notifications and this was not taken into consideration:
https://github.com/civicrm/civicrm-core/blob/4.6/css/civicrm.css#L3687

To fix this issue a special rule was added for tables inside notifications:

```scss
&.ui-notify-message-style {
  // ...

  table td,
  table th {
    background: $crm-white;
    border-color: $gray-light;
    color: $gray-dark;
  }
}
```

### CODE REVIEW

**Comment by jamienovick** [comment]
@reneolivo I think the table outlines shouldn't be shown at all? The SQL one for example isn't aligned correctly. best

**Comment by reneolivo** [comment]
Hello @jamienovick this table is defined in a CiviCRM template for a particular notification. If we hide the outline for this table we would hide outlines for all tables (inside of notifications of course).

Also the SQL issue is because in their template they have something like this:

```html
<tr><td><code>SQL Example....</code><td></tr>
```
<small>Notice the last &lt;td&gt; is missing a /. <a href="https://github.com/civicrm/civicrm-core/blob/4.6/templates/CRM/Contact/Import/Form/DataSource.hlp#L97">Source here</a></small>

I'll send them a PR to fix that typo.

**Comment by reneolivo** [comment]
Sent them a PR, doesn't follow any of their guidelines, but it's such a simple change, if they reject it they can implement it themselves:

https://github.com/civicrm/civicrm-core/pull/11698

**Comment by jamienovick** [comment]
ok got it. I hadnt realised we had actual tables in the help bubbles and thought this was just being used instead of divs...

File: scss/civicrm/common/_notifications.scss [hunk 1]
```diff
@@ -112,6 +114,13 @@
       line-height: 20px;
       margin-bottom: 10px;
     }
+
+    table td,
+    table th {
```
Comment (AkA84) [inline scss/civicrm/common/_notifications.scss]:
```scss
table {
  td,
  th {
    // ...
  }
}

**Review by AkA84 — CHANGES_REQUESTED** [review]

**Review by AkA84 — APPROVED** [review]

**Pull request merged**